examples/led_fade/README.md

```diff
@@ -0,0 +1,13 @@
+# LED Fade
+
+This project fades an LED when you turn a potentiometer knob.
+
+On the GrovePi+ or GrovePi Zero, connect a potentiometer to port A2 and a LED to port D3.
+
+If you're on Raspbian, trying this out by running:
```

Comment:
I only tested under Nerves.  I have never run grovepi from anything but nerves either. I was trying to follow the existing example.  I can update instructions for Nerves.    I will need to update my example to use grovepi from hex instead of the local folder.